packages/svelte/src/components/DataBrowserRouter/DataBrowserRouter.svelte

```diff
@@ -19,12 +19,9 @@
   }).initialize();
 
   let stateRef = writable(router.state);
-  let unsub = router.subscribe(stateRef.set);
+  router.subscribe(stateRef.set);
 
   setContext(RouterContextSymbol, { router, state: stateRef });
-  onDestroy(() => {
-    unsub();
```

Comment:
There is a `router.dispose` method you may want to call here ... and now I realize I'm also not calling in the `vue` side of things 🤦